sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -384,5 +322,55 @@ private void ValidateOperationStatus()
             if (!HasValue)
                 throw _requestFailedException;
```

Comment:
Should ValidateOperationStatus move onto `_operationInternal`?

Reply:
This is specific for Text Analytics. They have a unique method called `GetValues`, which uses this method.